k8s/manifest-templates/pharos.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pharos
+  labels:
+    name: pharos
+spec:
+  hostNetwork: true
+  containers:
+  - name: pharos-dashboard
+    image: ereslibre/pharos-dev:latest
```

Comment:
Here we could take two different ways:

1. Build the image locally first. The kubelet in this case would never pull any image, it would be tagged something like `pharos:development` and the first step would be to build the image locally with `docker build -t pharos:development .`.

2. Rely on a remote image. Not sure in which account should be (opensuse?) as `pharos-development` or `pharos:development`? In this case we could add an automated build once this PR gets merged so we have a Dockerfile to instruct the automated build where to look for.